### PR TITLE
update search proposal

### PR DESCRIPTION
**Pull request by carlory**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

Update the proposal based on the changes made on #1674 

**Which issue(s) this PR fixes**:
Part of  #1794 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/proposals/caching/README.md [hunk 1]
```diff
@@ -47,82 +47,76 @@ Goals:
...
+const (
+	// ResourceKindResourceRegistry is the name of the resource registry
+	ResourceKindResourceRegistry = "ResourceRegistry"
+	// ResourceSingularResourceRegistry is singular name of ResourceRegistry.
+	ResourceSingularResourceRegistry = "resourceregistry"
+	// ResourcePluralResourceRegistry is plural name of ResourceRegistry.
+	ResourcePluralResourceRegistry = "resourceregistries"
+	// ResourceNamespaceScopedResourceRegistry is the scope of the ResourceRegistry
+	ResourceNamespaceScopedResourceRegistry = false
+)
```
Comment (RainbowMango) [inline docs/proposals/caching/README.md:78]: Maybe we don't need to add the implemented details in the proposal.

**Review by RainbowMango — COMMENTED** [review]
Thanks @carlory for doing this.

I just talked to @liys87x about this task yesterday. Glad to see you helped it.

File: docs/proposals/caching/README.md (hunk 1 above)
Comment (carlory) [reply docs/proposals/caching/README.md]: @RainbowMango updated

**Comment by RainbowMango** [comment]
/approve

Leave LGTM to @liys87x

**Pull request merged**